versions.txt

```diff
@@ -2,16 +2,16 @@
 # by default with the OpenTelemetry Operator. This would usually be the latest
 # stable OpenTelemetry version. When you update this file, make sure to update the
 # the docs as well.
-opentelemetry-collector=0.102.1
+opentelemetry-collector=0.103.0
```

Comment:
i think we want to use collector [v0.103.1](https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/v0.103.1)

Reply:
Good catch.